matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/EventRelationsAggregationProcessor.kt

```diff
@@ -369,8 +343,7 @@ internal class EventRelationsAggregationProcessor @Inject constructor(
      * @param editions list of edition of event
      */
     private fun handleThreadSummaryEdition(
-            editedEvent: EventEntity?,
-            replaceEvent: TimelineEventEntity?,
+            editedEvent: EventEntity?, replaceEvent: TimelineEventEntity?,
```

Comment:
Maybe it is better to have on argument per line for consistency?

Reply:
dones